rpc/wrtc_client_stream.go

```diff
@@ -45,7 +50,14 @@ func newWebRTCClientStream(
 	stream *webrtcpb.Stream,
 	onDone func(id uint64),
 	logger utils.ZapCompatibleLogger,
-) *webrtcClientStream {
+) (*webrtcClientStream, error) {
+	// Assume that cancelation of the client channel's context means the peer
```

Comment:
This is a good start to a comment. But I feel like there's a bit of "what's the consequence of removing this"? This just looks like an optimization to inform the caller sooner that their invocation/stream isn't going to work.

Reply:
Added some elaboration there; let me know what you think. Thanks for asking for more, will be helpful for posterity.

Reply:
For sure; included (almost) exactly what you said there.